src/Symfony/Component/Messenger/Handler/HandlerInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Handler;
+
+/**
+ * Handlers can implements this interface. This allows them to be auto-configured and to
+ * handle multiple requests.
+ *
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface HandlerInterface
```

Comment:
`MessageHandlerInterface`? It's just a bit more obvious when you see the interface by itself in code (e.g. `FooHandler implements MessageHandlerInterface`

Reply:
It would match the tag name, it makes sense.